packages/core/src/sdJwt/sdJwt.ts

```diff
@@ -209,7 +205,7 @@ export class SdJwt<
      *
      */
     public withKeyBinding(
-        keyBinding: Jwt | KeyBinding | string
+        keyBinding: Jwt | KeyBinding<any, any> | string
```

Comment:
Was this change required?

Reply:
Yes, otherwise it will complain that `KeyBinding<Payload, Header>` is not compatible with `Keybinding<Record<string, unknown>, Record<string, unknown>>` (as that will be the default if you don't pass any)